lbry/blockchain/sync/synchronizer.py

```diff
@@ -43,6 +44,12 @@ def __init__(self, chain: Lbrycrd, db: Database):
         self.on_block_subscription: Optional[BroadcastSubscription] = None
         self.advance_loop_task: Optional[asyncio.Task] = None
         self.advance_loop_event = asyncio.Event()
+        self.filtering_channel_hashes = {
+            unhexlify(channel_id)[::-1] for channel_id in
+            os.getenv('FILTERING_CHANNEL_IDS', '').split(' ') if channel_id}
+        self.blocking_channel_hashes = {
+            unhexlify(channel_id)[::-1] for channel_id in
+            os.getenv('BLOCKING_CHANNEL_IDS', '').split(' ') if channel_id}
```

Comment:
New SDK uses the same config for client and server so these values should come from that config (which does support environment variables).